src/main/java/com/laytonsmith/core/events/drivers/EntityEvents.java

```diff
@@ -2153,4 +2154,67 @@ public boolean modifyEvent(String key, Mixed value, BindableEvent event) {
 			return false;
 		}
 	}
+
+	@api
+	public static class entity_unleash extends AbstractEvent {
+
+		@Override
+		public String getName() {
+			return "entity_unleash";
+		}
+
+		@Override
+		public String docs() {
+			return "{type: <macro> The type of entity | id: <macro> The UUID of entity}"
```

Comment:
The prefilters should be 'type' and 'reason'. The data should be 'id', 'type', and 'reason'.

Reply:
oh right, theres never been a uuid prefilter

Reply:
i think it clicked how this works, hopefully i just fixed it